changelog/add-5720-account-balances-card

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: add
+Comment: This change will not be visible to end-users. It is part of a larger project that will include a changelog entry when prepared.
```

Comment:
This is a changelog comment only, it will not appear in the user-facing changelog.

Reply:
TIL. I assumed it would only appear if the comment was empty.

Reply:
😝 I assume this to be true, I followed the instructions given by `npm run changelog`.

```
npm run changelog

> woocommerce-payments@5.5.1 changelog
> ./vendor/bin/changelogger add

Name your change file [default: add-5720-account-balances-card] >
Significance of the change, in the style of semantic versioning.
  [patch] Backwards-compatible bug fixes.
  [minor] Added (or deprecated) functionality in a backwards-compatible manner.
  [major] Broke backwards compatibility in some way.
 > patch
Type of change.
  [add   ] Add
  [fix   ] Fix
  [update] Update
  [dev   ] Dev
 > add
Changelog entry. May be left empty if this change is particularly insignificant.
 >
You omitted the changelog entry, which is fine. But please comment as to why no entry is needed.
 > This change will not be visible to end-users. It is part of a larger project that will include a changelog entry when prepared.
```

Reply:
Might be clearer to always say `na` instead of carefully crafting a throwaway sentence :)

Reply:
(na = not applicable)

Reply:
Also might be clearer to use `dev` rather than `add` when skipping 😁

Reply:
TBH, I read this prompt as: "prepare a carefully crafted throwaway sentence" 🥹

> You omitted the changelog entry, which is fine. But please comment as to why no entry is needed.

Reply:
I tested using `./vendor/bin/changelogger write` to write all changelog entries to `changelog.txt`.

Based on the list given `patch` `minor` `major` I would assume all would show in changelog.